src/pudl/output/ferc1.py

```diff
@@ -1632,6 +1719,104 @@ def initial_explosion_concatenation(
         )
         return exploded
 
+    def add_sizable_minority_corrections(
+        self: Self, exploded: pd.DataFrame
+    ) -> pd.DataFrame:
+        """Identify and fix the utilities that report calcs off-by one other fact.
+
+        We noticed that there are a sizable minority of utilities that report some
+        calculated values with a different set of child subcomponents. Our tooling
+        for these calculation expects all utilities to report in the same manner.
+        So we have identified the handful of worst calculable ``xbrl_factiod``
+        offenders :attr:`self.off_by_facts`. This method identifies data corrections
+        for those :attr:`self.off_by_facts` and adds them into the exploded data table.
+
+        The data corrections are identified by calculating the absolute difference
+        between the reported value and calculable value from the standard set of
+        subcomponents (via :func:`pudl.transform.ferc1.calculate_values_from_components`)
+        and finding the child factiods that have the same value as the absolute difference.
+        This indicates that the calculable parent factiod is off by that cooresponding
+        child fact.
+
+        Relatedly, :meth:`add_sizable_minority_corrections_to_calcs` adds these
+        :attr:`self.off_by_facts` to :attr:`self.exploded_calcs`.
+        """
+        if not self.off_by_facts:
+            return exploded
+        # add the abs_diff column for the calculated fields
+        calculated_df = pudl.transform.ferc1.calculate_values_from_components(
+            calculation_components=self.exploded_calcs[
+                ~self.exploded_calcs.is_within_table_calc
+                & ~self.exploded_calcs.is_total_to_subdimensions_calc
+            ],
+            data=exploded,
+            calc_idx=list(NodeId._fields),
+            value_col=self.value_col,
+        )
+
+        off_by = pd.DataFrame(self.off_by_facts)
+        # we calculate the non-abs diff here because many times the
+        # bad utility reporters include components that are not in the
+        # stock calculation so we need to remove it. the value of the
+        # correction record needs to be the raw diff.
+        not_close = (
+            calculated_df[calculated_df.abs_diff != 0]
```

Comment:
We use `np.isclose` elsewhere, why not here?

Reply:
ah good point.

in a previous commit i used the `is_not_close` from the ungrouped error frequency metric... but that felt like a little much setup wise so i reverted to a simpler selection here. but i'll conver to `~np.isclose` bc that is more in line with `is_not_close`/what we are doing in other places.

removed in [this commit](https://github.com/catalyst-cooperative/pudl/pull/3078/commits/b7328a05f9d414b22148e1de965713170e0413e2#diff-0c9cd108cbf26a3701799dcb4888a7d09d7c657eb9de859f4383b631e4042f7aL1742-L1747)

```python
       # add the ungrouped is_not_close column
        calculated_df["is_not_close"] = pudl.transform.ferc1.ErrorFrequency(
            by="ungrouped",
            is_close_tolerance=self.group_metric_checks.is_close_tolerance.error_frequency,
            metric_tolerance=self.group_metric_checks.group_metric_tolerances.ungrouped.error_frequency,
        ).is_not_close(calculated_df)
```